README.rst

```diff
@@ -0,0 +1,560 @@
+*********************************************
+
+.. contents:: :local:
+
+Introduction
+============
+
+This is the code repository for the GlanceSync component, the FIWARE Ops tool
+used to sincronise the glance images in the different Glance servers connected
+in the FIWARE Lab.
+
+This project is part of `FIWARE`_.
+
+Although this component has been developed for FIWARE, the software is highly
+configurable, do not have special requirements beyond OpenStack libraries and
+may be used with any other project or as a generic tool to synchronise images.
+Moreover, all the OpenStack interface is in a module and it is possible to
+adapt the code to support other platforms.
+
+Any feedback on this documentation is highly welcome, including bugs, typos
+or things you think should be included but are not. You can use 
+`github issues`_
+to provide feedback.
+
+Overall description
+===================
+
+Glancesync is a command line tool to solve the problem of the images
+synchronisation between regions. It synchronises glance servers in different
+regions taking the base of a master region. It was designed for FIWARE project,
+but it has been expanded to be useful for other users or projects.
+
+Sofware features
+----------------
+
+Glancesync synchronises all the images with certain metadata owned by a tenant
+from a master region to each other region in a federation (or a subset of them).
+This feature works out of the box without configuration. It requires only the
+same set of environment variables, which are needed to contact the
+keystone server, than the glance tool. It's also possible to set these
+parameters in a file instead of using environment variables.
+
+Glancesync synchronisation algorithm (i.e. the method to determine if a master
+image must be synchronised to the other regions) is configurable. By default
+all public images are synchronised, but it is enough with adding a line in the
+configuration file to synchronise only the public images with certain metadata.
+(e.g. federated_image=True)
+
+Glancesync supports also the synchronisation to regions which do not use the
+same keystone server than the master region and therefore require their own set
+of credentials. The regions are grouped by *targets*: two regions may be in the
+same *target* if they use the same credential (therefore, their glance servers
+are registered in the same keystone server). The only mandatory *target* is the
+``master`` target, where the master region is. Most of the Glancesync
+configuration, including the criteria to select which images are synchronised,
+is defined at target level. It is okay to create several targets using the same
+credential, for example if some regions only share a minimal set of images and
+others have a broader list.
+
+Glancesync by default does not replace existing images. If an
+image checksum is different between the region to synchronise and the master
+region, a warning is emitted. The user has the option of forcing the
```

Comment:
I don't see the ambiguity. In a user's manual, the user is always the user of the software, not the administrator of the remote node.

If you read in a browsesr documentation : "If the connection to the remote server fails because of the certificate, the user has the option of setting variable...." Do you thing that the user can refer to the webserver administrator?